### DISCUSSION TITLE
Is it possible to compile a Go program with TinyGo without jump tables please ?

### DISCUSSION THREAD

**Discussion by kajaaz** [tinygo-org/tinygo#4632] [Q&A]
Hello, 
I wondered if there is a similar command to `gcc -fno-jump-tables -o output_file source_file.c` to compile Go code without jump tables but only  with if/else statements (binary search) please ? 
Thank you

**Comment by aykevl** [comment]
Why do you need this feature? (Knowing why this is needed makes it much easier to help, or provide an alternative solution).

But to answer, no we don't have such a feature at the moment.
